src/reducer.js

```diff
@@ -11,43 +11,43 @@ function getHandlers({customHandlers, types, resourceName, pluralForm, allowedOp
   const {create, readOne, readMany, update, del} = allowedOperations;
 
   const createHandlers = create ? {
-    [`CREATE_${capitalResourceName}`]: defaultReducers.create,
-    [`CREATE_${capitalResourceName}_FAILURE`]: defaultReducers.createFailure,
-    [`CREATE_${capitalResourceName}_SUCCESS`]: defaultReducers.createSuccess,
-    [`CREATE_${capitalResourceName}_ABORTED`]: defaultReducers.createAborted,
-    [`CREATE_${capitalResourceName}_RESET_RESOLUTION`]: defaultReducers.createResetResolution
+    [`CREATE_${capitalResourceName}`]: defaultReducers.create.bind(null, idAttr),
```

Comment:
Mm, I'm not sure if I share the concern. I think bind will be faster than spreading. I'm going to keep this as-is for now, and we can look at perf metrics sometime to see if it makes sense to refactor to something else.